ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/service/RetentionModelExtractor.java

```diff
@@ -0,0 +1,45 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.view.service;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.RDFNode;
+import org.apache.jena.rdf.model.Statement;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.domain.constants.RdfConstants.RETENTION_TYPE;
+
+@Component
```

Comment:
Is there a need to have this a component? 
I see opportunities for static methods?

Reply:
I'm still in dubio on how the class will evolve when we have an ontology for fragmentations. For that reason I used a Component for the moment being.